src/main/java/wheresmymoney/command/ListCommand.java

```diff
@@ -24,7 +24,8 @@ private ArrayList<Expense> getExpensesToDisplay(ExpenseList expenseList) {
         }
     }
 
-    private void displayExpenses(ArrayList<Expense> expensesToDisplay, ExpenseList expenseList) {
+    private void displayExpenses(ArrayList<Expense> expensesToDisplay, ExpenseList expenseList)

```

Comment:
this function is not updated to display expenses with date - we should add that in

Reply:
corrected